testsuite/integration-arquillian/tests/base/src/test/resources/arquillian.xml

```diff
@@ -127,6 +127,7 @@
                 ${kie.maven.settings}
                 -Dauth.server.truststore=${auth.server.truststore}
                 -Dauth.server.truststore.password=${auth.server.truststore.password}
+                -Dauth.server.db.host=${auth.server.db.host}
```

Comment:
Why is this change necessary? Is it used in auth server at all? In such a case, there would be a change in XSLT updating server config.

If it is, there are many other places in this file where this has to be updated (every `auth-server-${auth.server}*`).

Reply:
Yes it is. Just like the other system properties for truststore. As you noticed, we are passing the DB host.

Will update other auth-server-*.

Reply:
@pedroigor Where exactly? I could not find auth.server.db.host anywhere in the xsl sheets and in keycloak.connectionsJpa.url it is AFAICT expanded.

Reply:
I still don't see where `auth.server.db.host` is used, even in the new commit (fc7c979). Can you point me to place where this is actually used within the auth server?